### PR TITLE
Adds dirt cheap surplus rifles to the nuke op uplink

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Gotta arm your reinforcements with something after all

:cl: Kor
rscadd: Nuke ops can purchase mosin nagants for 2 telecrystals.
/:cl:

### CODE REVIEW

**Comment by LanCartwright** [comment]
please change the mosin roundtype and buff it back to 60 damage

**Comment by Shapsy** [comment]
Yeah mosins need to do 60 to be viable, pls buff

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -202,6 +202,13 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	surplus = 25
 	include_modes = list(/datum/game_mode/nuclear)
 
+/datum/uplink_item/dangerous/bolt_action
+	name = "Surplus Rifle"
+	desc = "A horribly outdated bolt action weapon. You've got to be desperate to use this."
+	item = /obj/item/weapon/gun/projectile/shotgun/boltaction
```
Comment (neersighted) [inline code/modules/uplink/uplink_item.dm]: Any chance of a repath while you're in here?

**Comment by Zilenan91** [comment]
@shaps @LanCartwright making LSG literally a fucking SAW

35-45 would be good with some AP if we want it buffed, and the ammo should really be craftable.

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/modules/uplink/uplink_item.dm]: Why? It inherits the pump requirement of a shotgun

**Comment by Shapsy** [comment]
Then the issue is with LSG not the mosin itself

**Comment by KorPhaeron** [comment]
I can just change the bullet that LSG uses it is literally a non issue

**Pull request merged**